src/js/media-container.js

```diff
@@ -304,51 +304,66 @@ class MediaContainer extends window.HTMLElement {
     // This allows things like autoplay and programmatic playing to also initiate hiding controls (CJP)
     this.setAttribute('user-inactive', 'user-inactive');
 
-    const scheduleInactive = () => {
+    const setInactive = () => {
```

Comment:
nit (non-blocking): If we're going to model this as a property, we should probably just make it an actual property of the media-container.